public/app/brothers/brothers.js

```diff
@@ -0,0 +1,8 @@
+angular.module('KDRPoints')
```

Comment:
also style it like other controllers I have bc most of the things I do have a purpose